Maestro/subscriptions.json

```diff
@@ -1047,11 +1047,11 @@
         }
       }
     },
-    // Automate opening PRs to merge cli 3.1.3xx to release/5.0.1xx-preview3
+    // Automate opening PRs to merge cli 3.1.3xx to 3.1.4xx
     {
       "triggerPaths": [
-        "https://github.com/dotnet/sdk/blob/release/3.1.3xx/**/*",
-        "https://github.com/dotnet/installer/blob/release/3.1.3xx/**/*"
```

Comment:
Shouldn't this stay as 3.1.3xx? So we have 3.1.3xx-> 3.1.4xx?

Reply:
ah, ya we should disable that. 3.1.3xx will get the latest runtime soon